src/Symfony/Component/AssetMapper/Command/ImportMapInstallCommand.php

```diff
@@ -27,7 +29,8 @@
 final class ImportMapInstallCommand extends Command
 {
     public function __construct(
-        private readonly ImportMapManager $importMapManager,
+        private readonly RemotePackageDownloader $packageDownloader,
+        private readonly string $projectDir,
```

Comment:
Is this necessary "just" to render a success message ?

Reply:
It is... which is kind of lame, but otherwise, I'll be rendering the whole "Downloaded assets into /foo/bar/baz/my-project/assets/vendor"